src/main/kotlin/com/expedia/graphql/directives/KotlinDirectiveWiringFactory.kt

```diff
@@ -64,23 +59,15 @@ open class KotlinDirectiveWiringFactory(
             val directiveWiring = discoverWiringProvider(directive.name, env)
             if (directiveWiring != null) {
                 modifiedObject = directiveWiring.wireOnEnvironment(env)
-            } else {
-                throw InvalidSchemaDirectiveWiringException("No directive wiring provided for ${directive.name}")
```

Comment:
Is this exception used elsewhere? Can we delete it?

Reply:
its used few lines above

Reply:
🤦‍♂ Sorry, reviewing on mobile